to_spaces/app.py

```
@@ -0,0 +1,56 @@
+import asyncio
+import os
+from typing import Tuple
+
+import numpy as np
+import gradio as gr
+import google.genai as genai
```

Pylint (3.3.7)

[refactor] Use 'from google import genai' instead

(R0402)